packages/techdocs-cli-embedded-app/src/App.tsx

```diff
@@ -72,6 +73,7 @@ const routes = (
     >
       {techDocsPage}
       <TechDocsAddons>
+        <ReportIssue />
```

Comment:
We should discuss if all addons should be present in the CLI embedded app, and also if we want to add them to the main app directly as well

Reply:
Yup, will probably leave out for now, and we can re-add when we are ready.